configuration/wireless/access-point-bridged.md

```diff
@@ -0,0 +1,157 @@
+## Install the access point software
+
+In order to work as a bridged access point, the Raspberry Pi needs to have the `hostapd` access point software package installed:
+
+```
+sudo apt install hostapd
+```
+Enable the wireless access point service to start when your Raspberry Pi boots:
+
+```
+sudo systemctl unmask hostapd
+sudo systemctl enable hostapd
+```
+
+Software installation is complete. We will configure the access point software later on.
+
+<a name="bridging"></a>
+## Setup the network bridge
+
+A bridge network device in the Raspberry Pi will join the ethernet and wireless networks using its built-in interfaces.
+
+### Create a bridge device and populate the bridge
+
+Add a bridge network device named `br0` by creating the following file:
+
+```
+sudo nano /etc/systemd/network/bridge-br0.netdev
+
+[NetDev]
+Name=br0
+Kind=bridge
+```
+
+In order to bridge the ethernet network with the wireless network, first add the built-in ethernet interface (`eth0`) as bridge member with the following file:
+
+```
+sudo nano /etc/systemd/network/br0-member-eth0.network
+
+[Match]
+Name=eth0
+
+[Network]
+Bridge=br0
+```
+
+*Note:* The access point software will add the wireless interface `wlan0` to the bridge when the service starts. There is no need to create a file for that interface. This situation is particular to WiFi network interfaces.
+
+Now enable the `systemd-networkd` service to create and populate the bridge when your Raspberry Pi boots:
+
+```
+sudo systemctl enable systemd-networkd
+```
+
+### Define the bridge device IP configuration
+
+Network interfaces that are members of a bridge device are never assigned an IP address, since they communicate via the bridge. The bridge device itself needs an IP address, so that you can reach your Raspberry Pi on the network.
+
+`dhcpcd`, the DHCP client on the Raspberry Pi, automatically requests an IP address for every active interface. So we need to block the `eth0` and `wlan0` interfaces from being processed, and let `dhcpcd` only configure `br0` via DHCP.
+
+```
+sudo nano /etc/dhcpcd.conf
+```
+
+Add the following line near the beginning of the file (above the first `interface xxx` line, if any):
+```
+denyinterfaces wlan0 eth0
+```
+Interface `br0` will be configured as per defaults via DHCP, no specific entry is necessary. Save the file to complete the IP configuration of the machine.
+
+<a name="ap-config"></a>
+## Configure the access point software
+
+Create the `hostapd` configuration file, located at `/etc/hostapd/hostapd.conf`, to add the various parameters for your wireless network. 
+
+```
+sudo nano /etc/hostapd/hostapd.conf
+```
+
+Add the information below to the configuration file. This configuration assumes we are using channel 7, with a network name of `NameOfNetwork`, and a password `AardvarkBadgerHedgehog`. Note that the name and password should **not** have quotes around them. The passphrase should be between 8 and 64 characters in length.
```

Comment:
I've not followed this setup myself, but is it worth suggesting that the `NameOfNetwork` is different to any existing wireless network names?

Reply:
Sort answer: no I don't think it is useful.
I haven't touched the wireless setup part of the original doc, mostly.
I do not think there is a requirement for the SSID to be unique, in case of multiple identical network names (with identical password) a client should be able to roam with more or less success, depending on client, signals strengths...
Of course if people start connecting to the wrong AP they might get confused by what they see. But the number of users of this tutorial having multiple APs is probably low anyway.

Reply:
> Of course if people start connecting to the wrong AP they might get confused by what they see.

I'm imagining that most people following this tutorial will probably be home users with an existing wireless network being served from their broadband router.
But maybe I'm just being over-cautious and it's obvious that the network-name they choose for their RPi AP should be different from their router's AP? :man_shrugging: